### DISCUSSION TITLE
Add animations to Beatmap "configuration" buttons

### DISCUSSION THREAD

**Discussion by greenmg2003** [ppy/osu#17016] [Ideas] [answered]
I'm sorry if my vocabulary isn't that good. What I mean is to add animations to these buttons (maybe triangle animation like the animation you used in the "ruleset switching tab"?)
![image](https://user-images.githubusercontent.com/65149680/155921310-5707a076-0895-440d-a64d-bbc44ca17a81.png)
Or maybe these buttons:
![image](https://user-images.githubusercontent.com/65149680/155921707-7baebae2-40ba-4c49-a878-305179d510b0.png)

**Comment by smoogipoo** [comment, marked as answer]
The entire screen is being redesigned https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0%3A1
